orga/app/components/manage-authentication-method-modal.js

```diff
@@ -73,8 +73,11 @@ export default class ManageAuthenticationMethodModal extends Component {
     try {
       await schoolingRegistrationDependentUser.save({ adapterOptions: { generateUsernameAndTemporaryPassword: true } });
       this.args.student.username = schoolingRegistrationDependentUser.username;
-      this.generatedPassword = schoolingRegistrationDependentUser.generatedPassword;
-      this.isUniquePasswordVisible = !this.isUniquePasswordVisible;
+
+      if (this.args.student.email === undefined || this.args.student.email === null) {
```

Comment:
```suggestion
      if (!this.args.student.email) {
```
Voir https://developer.mozilla.org/fr/docs/Glossaire/Truthy.

Reply:
done !